tests/components/august/test_binary_sensor.py

```diff
@@ -90,6 +95,29 @@
     assert binary_sensor_tmt100_name_ding.state == STATE_UNAVAILABLE
 
 
+async def test_create_doorbell_with_motion(hass):
+    """Test creation of a doorbell."""
+    doorbell_one = await _mock_doorbell_from_fixture(hass, "get_doorbell.json")
+    doorbell_details = [doorbell_one]
+    activities = await _mock_activities_from_fixture(
+        hass, "get_activity.doorbell_motion.json"
+    )
+    await _create_august_with_devices(hass, doorbell_details, activities=activities)
+
+    binary_sensor_k98gidt45gul_name_motion = hass.states.get(
+        "binary_sensor.k98gidt45gul_name_motion"
+    )
+    assert binary_sensor_k98gidt45gul_name_motion.state == STATE_ON
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5072309)